.github/actions/setup/action.yaml

```diff
@@ -1,6 +1,14 @@
 name: Setup Environment
 description: UDS Environment Setup
 
+inputs:
+  username:
+    description: Registry1 user
+    required: true
+  password:
+    description: Registry1 PW
```

Comment:
small nit: Can we just write out password? I get that the description is pretty much already redundant 😄

Reply:
Done!